test/Table_Tests/src/In_Memory/Table_Conversion_Spec.enso

```diff
@@ -206,6 +206,20 @@ add_specs suite_builder =
             table = Table.new [["aaa", [1, 2]], ["bbb", [Date.new 2020 12 1 . up_to (Date.new 2020 12 3), Date.new 2022 12 1 . up_to (Date.new 2022 12 2)]]]
             expected = Table.new [["aaa", [1, 2]], ["bbb 0", [Date.new 2020 12 1, Date.new 2022 12 1]], ["bbb 1", [Date.new 2020 12 2, Nothing]]]
             table.expand_column "bbb" . should_equal expected
+        
+        group_builder.specify "Expands rows within a column" <|
+            table1 = Table.new [["aaa", [1, 2, 3]], ["bbb", [4, 5, 6]], ["ccc", [7, 8, 9]]]
+            table2 = Table.new [["aaa", [10]], ["bbb", [11]], ["ccc", [12]]]
+            table = Table.new [["tables", [table1, table2]]]
+            expected = Table.new [["tables aaa", [1, 2, 3, 10]], ["tables bbb", [4, 5, 6, 11]], ["tables ccc", [7, 8, 9, 12]]]
+            table.expand_to_rows "tables" . expand_column "tables" . should_equal expected
```

Comment:
shall we have an example where the tables do not have the same sets of columns?

Reply:
Done